schema.asn

```diff
@@ -465,3 +467,33 @@ BEGIN
       ...
    }
 END
+
+Zjpm-Extension
+DEFINITIONS
+   AUTOMATIC TAGS ::=
+BEGIN
+   EXPORTS Zjpm;
+
+   Zjpm ::= SEQUENCE {
+      -- 1 bit to indicate whether the pointer masking extension is present.
+      -- If this extension is included, it's assumed pointer masking is present.
+
+      -- 1 bit to indicate whether pointer masking for instructions is supported.
+      pmInstruction BOOLEAN DEFAULT FALSE,
+
+      -- 4 bits (1 bit per privilege level) to indicate whether pointer masking
+      -- is supported at that privilege level.
+      mSupported BOOLEAN DEFAULT FALSE,
+      sSupported BOOLEAN DEFAULT FALSE,
+      vSupported BOOLEAN DEFAULT FALSE,
+      uSupported BOOLEAN DEFAULT FALSE,
+
+      -- 5/6/7 bits (for RV32/64/128) to indicate how many bits from the top are
+      -- maskable.
+      maskBitcount INTEGER (1..128),
+
```

Comment:
Is that how many continuous bits are maskable?  that could be 5, 6, or 7 bits? Means 3-bits is enough to indicate how many bits can be masked? Or the maskable bits could be not continuously?

Reply:
That's a good question. I thought Martin meant 5 bits are required on RV32 to encode values from 1 to 32, and so on. I won't merge this until somebody familiar with the spec has taken a look at it.

Reply:
The definition is that fields maskBitcount..31 (or 63, 127) are maskable for RV32 (or RV64, RV128). This choice means that maskBitcount = 0 is the natural way of encoding that the entire pointer word is maskable (and there is no way to encode that no bits are maskable, which would not be valid according to the spec). Maybe we should call the field "maskableBits" instead of maskBitcount and add this explanation in a comment.

Looks good to me otherwise.

Reply:
How about we call this lowestMaskBit?

Reply:
SGTM